### ISSUE TITLE
small feature request: provide feedback while .deb files are being installed

### ISSUE THREAD

**Issue by robinpaulson** [pimlie/ubuntu-mainline-kernel.sh#43]
Currently the script lets me know it's downloading the .deb files, by updating a number indicating percentage completion.

When it comes to installation, everything is silent, I've wondered a couple of times if anything is happening (my machine is relatively old, so it takes a while to install). I'm not after anything flash, a similar increasing % indicator is fine. Of course, this is not at all high priority, more a nice-to-have

Cheers

**Comment by pimlie** [comment]
Yeah, have noticed this as well. My pc is not that old but am using some dkms modules which also take some time, some feedback would probably be nice.

**Comment by robinpaulson** [comment]
I didn't even think about DKMS; I've got three of them which will add a chunk of time to the process.

**Comment by robinpaulson** [comment]
It wouldn't even need to be a % indicator, they're notoriously inaccurate anyway: a spinning thingy would be great.

**Comment by gothicVI** [comment]
You could run the script verbosely (using the `-d` option) in order to get more output.
